src/AppKit/NSGraphics.cs

```diff
@@ -152,11 +156,11 @@ public static void FrameRect (CGRect rect, nfloat frameWidth, NSCompositingOpera
 		}
 
 		[DllImport (Constants.AppKitLibrary, EntryPoint = "NSShowAnimationEffect")]
-		extern static void NSShowAnimationEffect (nuint animationEffect, CGPoint centerLocation, CGSize size, NSObject animationDelegate, Selector didEndSelector, IntPtr contextInfo);
+		extern static void NSShowAnimationEffect (nuint animationEffect, CGPoint centerLocation, CGSize size, NativeHandle animationDelegate, NativeHandle didEndSelector, IntPtr contextInfo);
 
 		public static void ShowAnimationEffect (NSAnimationEffect animationEffect, CGPoint centerLocation, CGSize size, NSObject animationDelegate, Selector didEndSelector, IntPtr contextInfo)
 		{
-			NSShowAnimationEffect ((nuint) (ulong) animationEffect, centerLocation, size, animationDelegate, didEndSelector, contextInfo);
+			NSShowAnimationEffect ((nuint) (ulong) animationEffect, centerLocation, size, animationDelegate.Handle, didEndSelector.Handle, contextInfo);
```

Comment:
This avoids potential NullReferenceExceptions:

```suggestion
			NSShowAnimationEffect ((nuint) (ulong) animationEffect, centerLocation, size, animationDelegate.GetHandle (), didEndSelector.GetHandle (), contextInfo);
```